spec/text_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'text'
+
+describe Text do
+
+  it 'initialize raises error when missing a number' do
+    expect{described_class.new()}.to raise_error(ArgumentError)
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.